tracks/ruby/exercises/word-count.md

```diff
@@ -0,0 +1,55 @@
+#### Intro
+Side exercise. Great to step up from basic loops and `Enumerable#each` to more powerful Enumberable methods. 
+
+#### Reasonable solutions
+```
+WORD_REGEX = /\b[\w|']+\b/
+
+  def initialize(phrase)
+    @words = phrase
```

Comment:
I love this solution - it's very close to what I did myself 🙌 I think we should use `@phrase` here instead of `@words`, though. Thoughts?

Reply:
I don't know 😄 That's a bit of a controversial subject from what I saw in a recent Slack discussion. Probably not?